app/forms/steps/capital/partner_trust_fund_form.rb

```diff
@@ -10,12 +10,16 @@ class PartnerTrustFundForm < Steps::BaseFormObject
 
       validates :partner_will_benefit_from_trust_fund, inclusion: { in: YesNoAnswer.values }
 
-      validates(
-        :partner_trust_fund_amount_held,
-        :partner_trust_fund_yearly_dividend,
-        presence: true,
-        if: -> { partner_will_benefit_from_trust_fund&.yes? }
-      )
+      validates :partner_trust_fund_amount_held, numericality: {
+        greater_than: 0,
+        less_than_or_equal_to: 99_999_999.99
+      }, if: -> { partner_will_benefit_from_trust_fund&.yes? }
+
+      validates :partner_trust_fund_yearly_dividend, numericality {
+        greater_than_or_equal_to: 0,
+        less_than_or_equal_to: 99_999_999.99
```

Comment:
This one isn't on the MAAT spreadsheet, but it does get combined with all the income payments that we inject into 'Other income' so seemed sensible to restrict, even if not a very scientific way (i.e. if anything else is added to it it will break) - feel free to reduce if you feel it's necessary